{"name": "页面2-标题1", "status": "failed", "statusDetails": {"message": "AssertionError: assert 5 == 2", "trace": "self = <test_pytest1.Test object at 0x0000000003871648>, num1 = 5\n\n    @pytest.mark.parametrize('num1', [6, 5, 7])\n    @allure.story('页面2')\n    @allure.title('页面2-标题1')\n    def test_bbc(self, num1):\n>       assert num1 == 2\nE       assert 5 == 2\n\ntest_pytest1.py:18: AssertionError"}, "parameters": [{"name": "num1", "value": "5"}], "start": 1603329949805, "stop": 1603329949805, "uuid": "b7923a49-22de-4661-b957-bafb0cd0ea7e", "historyId": "cd86c05fa31ae778ddd258508bf39614", "testCaseId": "7ca0b45487ca02e4a73c77eb0de43b00", "fullName": "test_pytest1.Test#test_bbc", "labels": [{"name": "story", "value": "页面2"}, {"name": "feature", "value": "模块1"}, {"name": "suite", "value": "test_pytest1"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "huang-PC"}, {"name": "thread", "value": "193688-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest1"}]}